bin/hdfcoinc/pycbc_make_psd_estimation_workflow

```diff
@@ -109,7 +109,8 @@ for ifo, segments in analyzable_segs.items():
             all_broken_segments.append(s)
             start_time = end_time
 
-            if len(partial_broken_segments) >= max_segs_per_job:
+            if (len(partial_broken_segments) >= max_segs_per_job) \
+                    or (start_time + psd_job_length + pad_data > seg[1]):
```

Comment:
What exactly is the intention here in terms of parallelization? Previously I had understood that each job would analyse max_segs_per_job * psd_job_length. Now it seems that at the end of every segment a new job will be created. So previously all jobs would have the same run time, but now there will be quite a lot of inbalance. A lot of jobs looking at short analysis segments will run very quickly (potentially containing 1 or a small number of "broken segments"). This will also mean a lot more PSD jobs will be created, potentially leading to some I/O load when all of the jobs run at the same time .... I think the previous approach, where all jobs have the same run time was better!

However, I can see that the *last* group of broken segments would not have been analysed as there was no code to create a final job to analyse the < max_segs_per_job segments at the end!

Also is it possible to change "broken" -> "split". "Broken" implies not working and worried me when I saw this in an output file name!